sdk/maps/Azure.Maps.Search/src/MapsSearchClient.cs

```diff
@@ -1134,274 +472,40 @@ public virtual async Task<ReverseSearchAddressBatchOperation> ReverseSearchAddre
         }
 
         /// <summary>
-        /// The Search Address Reverse Batch API sends batches of queries to <see href="https://docs.microsoft.com/rest/api/maps/search/getsearchaddressreverse">Search Address Reverse API</see> using just a single API call. You can call Search Address Reverse Batch API to run either asynchronously (async) or synchronously (sync). The async API allows caller to batch up to **10,000** queries and sync API up to **100** queries.
+        /// The Reverse Geocoding Batch API sends batches of queries to [Reverse Geocoding API](/rest/api/maps/search/get-reverse-geocoding) using just a single API call. The API allows caller to batch up to <c>100</c> queries.
+        ///
+        /// A _reverse geocoding_ batchItem object can accept any of the supported _reverse geocoding_ [URI parameters](/rest/api/maps/search/get-reverse-geocoding#uri-parameters).
```

Comment:
1. Remove underscore, for "reverse geocoding" because it's not supported in docstring
2. Use `<see href="xxx">yyy</see>` for hyperlink

Reply:
@dubiety done